lib/srasConfig.js

```diff
@@ -2,16 +2,26 @@ const parseSRASConfig = (data) => {
   data = data || {};
 
   const rules = data.rules || {};
+  const jsRules = rules.js || {};
   const imgRules = rules.images || {};
   const audRules = rules.audio || {};
   const requiredTypes = [
     ...Object.keys(imgRules).filter((i) => !!imgRules[i].require),
     ...Object.keys(audRules).filter((i) => !!audRules[i].require),
   ];
-  const ignored = new RegExp((data.ignore || []).join('|'));
-  const unpermitted = new RegExp((data.unpermitted || []).join('|'));
+
+  let ignored = /$doesntmatch/;
```

Comment:
This line can be written to allow `ignore` to remain a const:

`const ignored = data.ignore ? /$doesntmatch/ : new RegExp(data.ignore.join('|'));`

Reply:
Good idea. I'll do that.